FlaskApp/app/static/js/script_barchart.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by iris on 2/20/16.
+ */
+$(function () {
+    $('#container').highcharts({
```

Comment:
@hyperchi agree. The id is unique in its scope, since two scripts is applied for two different pages, but yeah, it's a good suggestion to make it less confusing.